app/src/layouts/organizationsControl/organizationsControl.jsx

```diff
@@ -45,12 +45,18 @@ export const OrganizationsControl = ({
             'instance-level': isExtendedNav,
           })}
         >
-          {!isExtendedNav ? <i className={cx('arrow-icon')}>{Parser(ArrowLeftIcon)}</i> : null}
-          <Link to={link} className={cx('organization-btn')} onClick={closeSidebar}>
-            <div className={cx('top-title')}>{titles.topTitle}</div>
-          </Link>
+          {isExtendedNav ? (
```

Comment:
Could you please reverse the `isExtendedNav` boolean value come from upper components?
I mean that when it is `isExtendedNav=true`, the content should be **extended** with smth, but right now with this option the content is shorter than without it.

Reply:
Done

Reply:
Done